frontend/smart-kickers-game/src/App.js

```diff
@@ -4,39 +4,35 @@ import {Button} from "./components/button"
 
 function App() {
 
-  let [blueScore, setBlueScore] = useState(0);
-  let [whiteScore, setWhiteScore] = useState(0);
+  const [blueScore, setBlueScore] = useState(0);
+  const [whiteScore, setWhiteScore] = useState(0);
 
   useEffect(() => {
     const socket = new WebSocket('ws://localhost:3006/csc');
     //const socket = new WebSocket("ws://localhost:3000")
     socket.onopen = function () {
-      console.log("connected");
       //send to server
       socket.send("Hello from client")
       socket.onmessage = (msg) => {
-        console.log(msg);
         msg = JSON.parse(msg.data)
-        console.log(msg)
         setBlueScore(msg.blueScore)
         setWhiteScore(msg.whiteScore)
-     
-        console.log(msg);
       };
     };
   });
 
   return (
     <>
       <h1>Smart Kickers</h1>
-      <div className="game-result-container" data-testid="blue-team-score">
-        Blue: {blueScore}
-          {"   "}
-        White: {whiteScore}
+      <div className="game-result-container">
+        <p className="game-result-item">Blue: {blueScore}</p>
+        <p className="game-result-item">White: {whiteScore}</p>
       </div>
-      <center><Button >Reset game</Button></center>
+      <center>
+        <Button>Reset game</Button>
+      </center>
     </>
-  )
+  );
 }
 
 export default App
```

Comment:
EOF

Reply:
> I have small talk with @Wawrzyn321 and @Lyczeq and we all agree that the whole team has to configure prettier. So please add in this PR config for prettier in `.prettierrc` file. You can check our config from previous TeamHack (it's not the best, but works) [Kyma-showcase prettierrc](https://github.com/kyma-incubator/kyma-showcase/blob/main/frontend/.prettierrc)

Config added with adjustment for current prettier version (semicolon become semi), and files fixed with it.